cmd/template/main.go

```diff
@@ -48,6 +48,28 @@ clean:
 .PHONY: all build run test clean
 		`)
 }
+func GitIgnoreTemplate() []byte {
+	return []byte(
+		`
+# Binaries for programs and plugins
+*.exe
+*.exe~
+*.dll
+*.so
+*.dylib
+
+# Test binary, built with "go test -c"
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Dependency directories (remove the comment below to include it)
+# vendor/
+
+# Go workspace file
+go.work
+		`)
```

Comment:
You are missing a closing } bracket on this function that is causing 
```go build``` to error and fail out